frontend/src/models/eventDefinitionsModel.ts

```diff
@@ -23,7 +23,7 @@ export const eventDefinitionsModel = kea<eventDefinitionsModelType>({
             {
                 loadEventDefinitions: async (initial?: boolean) => {
                     const url = initial
-                        ? `api/projects/${teamLogic.values.currentTeamId}/event_definitions/?limit=5000`
+                        ? `api/projects/${teamLogic.values.currentTeamId}/event_definitions/?limit=5000&event_type=event`
```

Comment:
why doesn't event definitions only return events why do we have to add the event type? 😮

Reply:
Good call, I'll have this endpoint default to returning only events